invenio_rdm_records/handlers.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
```

Comment:
Can you move this file to ``resources/iiif_handler.py``? It belongs to presentation layer (accessing service and g object).

Reply:
discuss IRL, moved to `iiif.py`